test/e2e/common_test.go

```diff
@@ -291,15 +294,41 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	}
 	if remote {
 		p.PodmanTest.RemotePodmanBinary = podmanRemoteBinary
-		uuid := stringid.GenerateNonCryptoID()
+		var pathPrefix string
 		if !rootless.IsRootless() {
-			p.RemoteSocket = fmt.Sprintf("unix:/run/podman/podman-%s.sock", uuid)
+			pathPrefix = "/run/podman/podman"
 		} else {
 			runtimeDir := os.Getenv("XDG_RUNTIME_DIR")
-			socket := fmt.Sprintf("podman-%s.sock", uuid)
-			fqpath := filepath.Join(runtimeDir, socket)
-			p.RemoteSocket = fmt.Sprintf("unix:%s", fqpath)
+			pathPrefix = filepath.Join(runtimeDir, "podman")
```

Comment:
every integration test has it's own dedicated `runtimeDir`, so this change alone will guarantee collisions are impossible.

Reply:
This wasn’t intended to change anything about the directory where the socket is placed; did I make a mistake?

Reply:
oops, sorry, I thought this was runroot.

Reply:
though to be clear...using the runroot for the socket would eliminate the possibilities of clashes.  It also might run into the socket filename length limitation.